pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -279,18 +279,25 @@ func GetOperInfo() (info.DataSecAtRestStatus, string) {
 			//fscrypt is setup, but not being used
 			log.Debug("Setting status to Error")
 			return info.DataSecAtRestStatus_DATASEC_AT_REST_ERROR,
-				"Disk Encryption Initialization failure"
+				"Initialization failure"
 		} else {
 			//fscrypt is setup, and being used on /persist
 			log.Debug("Setting status to Enabled")
 			return info.DataSecAtRestStatus_DATASEC_AT_REST_ENABLED,
-				"Disk Encryption is Active"
+				"Encrypted File Stores: Config, App Image"
 		}
 	} else {
-		//This is due to ext3 partition
-		log.Debug("Setting status to Disabled")
-		return info.DataSecAtRestStatus_DATASEC_AT_REST_DISABLED,
-			"File system is incompatible - Needs a disruptive upgrade"
+		if !tpmmgr.IsTpmEnabled() {
+			//This is due to ext3 partition
+			log.Debug("Setting status to disabled, HSM is not in use")
+			return info.DataSecAtRestStatus_DATASEC_AT_REST_DISABLED,
+				"HSM is either absent or not in use"
+		} else {
+			//This is due to ext3 partition
+			log.Debug("setting status to disabled, ext3 partition")
+			return info.DataSecAtRestStatus_DATASEC_AT_REST_DISABLED,
+				"file system is incompatible, needs a disruptive upgrade"
```

Comment:
Why did you downcase the leading "file" when all the other messages start with a leading capital letter?
Yes, a nit, but so was your PR ;-)

Reply:
Addressed your comment.